pkg/bot/mattermost.go

```diff
@@ -151,7 +151,8 @@ func (mm *mattermostMessage) handleMessage(b MMBot) {
 	if channelType == mmChannelPrivate || channelType == mmChannelPublic {
 		// Message posted in a channel
 		// Serve only if starts with mention
-		if !strings.HasPrefix(post.Message, "@"+b.BotName+" ") {
+		if !strings.HasPrefix(post.Message, fmt.Sprintf("@%s ", b.BotName)) &&
+			!strings.HasPrefix(post.Message, fmt.Sprintf("@%s ", strings.ToLower(b.BotName))) {
```

Comment:
Does mattermost ignore case for user names? Does users with upper case username and smaller case belong to the same user?

Reply:
Generally yes, Mattermost allows users to create username with upper case characters, and it converts them to lowercase - please see discussion from https://github.com/infracloudio/botkube/issues/618 and this comment: https://github.com/mattermost/mattermost-server/issues/7921#issuecomment-348877595

Reply:
got it, thanks